extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js

```diff
@@ -222,7 +223,7 @@ angular.module('PaperUI.controllers.control', []).controller('ControlPageControl
         'SoundVolume' : {
             label : 'Volume',
             icon : 'volume_up',
-            hideSwitch : true
+            showSwitch : false
```

Comment:
Yes, if we remove this line the toggle will not be displayed in the same way as if we set it to false.